subsys/mpsl/cx/Kconfig

```diff
@@ -4,26 +4,26 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-DT_COMPAT_CX_NRF700X := nordic,nrf700x-coex
+DT_COMPAT_CX_NRF700X := nordic,nrf70-coex
 DT_COMPAT_CX_SOFTWARE := nordic,software-coex
 
 config MPSL_CX_ANY_SUPPORT
 	bool
-	default $(dt_nodelabel_enabled,nrf_radio_coex)
+	default $(dt_nodelabel_enabled,nrf70_1)
```

Comment:
what is this new node `nrf70_1`? Can you please share some reference?

Reply:
The node in NCS 2.7.0 dts files were labelled as nrf_radio_coex and these were part of both cpunet.dts and common.dtsi as given below.  

boards\nordic\nrf7002dk\nrf7002dk_nrf5340_cpunet.dts
                **nrf_radio_coex**: nrf7002-coex {
                                status = "okay";
                                compatible = "nordic,nrf700x-coex";
                                req-gpios = <&gpio0 28 GPIO_ACTIVE_HIGH>;
                                status0-gpios = <&gpio0 30 GPIO_ACTIVE_HIGH>;
                                grant-gpios = <&gpio0 24 (GPIO_PULL_DOWN | GPIO_ACTIVE_LOW)>;
                                swctrl1-gpios = <&gpio0 29 GPIO_ACTIVE_HIGH>;
                };

boards\nordic\nrf7002dk\nrf5340_cpuapp_common.dtsi
                **nrf_radio_coex**: nrf7002-coex {
                                status = "okay";
                                compatible = "nordic,nrf700x-coex";
                                req-gpios = <&gpio0 28 GPIO_ACTIVE_HIGH>;
                                status0-gpios = <&gpio0 30 GPIO_ACTIVE_HIGH>;
                                grant-gpios = <&gpio0 24 (GPIO_PULL_DOWN | GPIO_ACTIVE_LOW)>;
                                swctrl1-gpios = <&gpio0 29 GPIO_ACTIVE_HIGH>;
                                srrf-switch-gpios = <&gpio1 10 GPIO_ACTIVE_HIGH>;
                };

When the recent upstream changes were done, nrf_radio_coex is renamed as nrf70. But, the node information is included only in nrf5340_cpuapp_common.dtsi and removed from the nrf5340_cpuapp_common.dtsi. 
With this change, the MPSL coex configuration parameters are not set as expected and so I added this node information back in the nrf5340_cpuapp_common.dtsi and initially used nrf70 as node label. But, this is throwing build errors as the same node label is available in nrf7002dk_nrf5340_cpuapp.dts as given below.

&qspi {
	nrf70: nrf7002@1 {
		compatible = "nordic,nrf7002-qspi";
		status = "okay";
		reg = <1>;
		qspi-frequency = <24000000>;
		qspi-quad-mode;

		#include "nrf70_common.dtsi"
		#include "nrf70_common_coex.dtsi"
		#include "nrf70_common_5g.dtsi"
	};
};

So, used nrf_70_1 as the node label to avoid build errors and get the MPSL coex configuration parameters as expected.

Reply:
can you format the code properly there because it looks to randomly have includes in it? Also how will this work for customers with out of tree boards from 2.7?

Reply:
> But, the node information is included only in nrf5340_cpuapp_common.dtsi and removed from the nrf5340_cpuapp_common.dtsi.

I guess you meant `included only in nrf7002dk_nrf5340_cpunet.dts` ? This was intentional, AFAIK these co-ex signals are used only by BLE, so, assigned to CPUNET, CPUAPP doesn't need this. @bjarki-andreasen .

Reply:
Note that aside from out-of-tree boards, there are multiple users of this node in the SDK as well, e.g.:
- https://github.com/nrfconnect/sdk-nrf/blob/a57c259d7bcf8770cb73343ab9334e7c9ba0459c/subsys/mpsl/cx/1wire/mpsl_cx_1wire.c#L29
- https://github.com/nrfconnect/sdk-nrf/blob/6041039b834ecadf433bf46660d483d5906d1098/subsys/mpsl/cx/3wire/mpsl_cx_3wire.c#L33
- https://github.com/nrfconnect/sdk-nrf/blob/cef6e6d9ce5ac2ec6020984fc2e82e6b97b1e086/samples/bluetooth/radio_coex_1wire/boards/nrf52840dk_nrf52840.overlay#L11

Reply:
> > But, the node information is included only in nrf5340_cpuapp_common.dtsi and removed from the nrf5340_cpuapp_common.dtsi.
> 
> I guess you meant `included only in nrf7002dk_nrf5340_cpunet.dts` ? This was intentional, AFAIK these co-ex signals are used only by BLE, so, assigned to CPUNET, CPUAPP doesn't need this. @bjarki-andreasen .

Yes Chaitanya, I meant included in only nrf7002dk_nrf5340_cpunet.dts.

I had a look at nrf7002dk_nrf5340_cpuapp.dts now. This also contains the node nrf70 and coex DTS included as given below.

&qspi {
	**nrf70**: nrf7002@1 {
		compatible = "nordic,nrf7002-qspi";
		status = "okay";
		reg = <1>;
		qspi-frequency = <24000000>;
		qspi-quad-mode;

		#include "nrf70_common.dtsi"
		**#include "nrf70_common_coex.dtsi"**
		#include "nrf70_common_5g.dtsi"
	};
};

Reply:
Look at this PR https://github.com/zephyrproject-rtos/zephyr/pull/79606